themes/default/data/awards.toml

```diff
@@ -14,7 +14,7 @@ url = "https://www.redpoint.com/infrared/100/"
 title = "Top Open Source Startups, 2023"
 featured = true
 img = "/logos/press/awards/traxcn-logo.png"
-url = "https://tracxn.com/d/emerging-startups/top-open-source-startups-2023"
+# url = "https://tracxn.com/d/emerging-startups/top-open-source-startups-2023"
```

Comment:
I'd just delete these; there's no need to keep them commented.

```suggestion
```

Reply:
Ok